apidoc/Titanium/UI/Attribute.yml

```diff
@@ -115,3 +124,119 @@ properties:
+    description: |
+      This property contains the distance (in points) from the leading margin 
+      of a text container to the beginning of lines other than the first. 
+      This value is always nonnegative float value.
+    type: Number
+
+  - name: tailIndent
+    summary: The trailing indentation of the receiver.
+    description: |
+      If positive, this value is the distance from the leading margin 
+      (for example, the left margin in left-to-right text). If 0 or negative, 
+      it is the distance from the trailing margin. This is a float value.
+    type: Number
+
+  - name: lineBreakMode
+    summary: The mode that should be used to break lines in the receiver.
+    description: |
+      This property contains the line break mode to be used laying out the text of paragraph.
+    type: Number
+    constants: [ Titanium.UI.ATTRIBUTE_LINE_BREAK_* ]
+
+  - name: maximumLineHeight
+    summary: The maximum line height of receiver.
+    description: |
+      This property contains the maximum height in points that any line in the receiver will occupy, 
+      regardless of the font size or size of any attached graphic. This value is always nonnegative. 
+      The default value is 0.
+    type: Number
+
+  - name: minimumLineHeight
+    summary: The minimum height of receiver.
+    description: |
+      This property contains the minimum height in points that any line in the receiver will occupy, 
+      regardless of the font size or size of any attached graphic. This value must be nonnegative.
+    type: Number
+
+  - name: lineSpacing
+    summary: The distance in points between the bottom of one line fragment and the top of the next.
+    description: |
+      This value is always nonnegative. This value is included in the line fragment heights 
+      in the layout manager.
+    type: Number
+
+  - name: paragraphSpacing
+    summary: The space after the end of the paragraph.
+    description: |
+      This property contains the space (measured in points) added at the end of the paragraph 
+      to separate it from the following paragraph. This value must be nonnegative. The space 
+      between paragraphs is determined by adding the paragraphSpacing of previous paragraph and 
+      the paragraphSpacingBefore of current paragraph.
+    type: Number
+
+  - name: paragraphSpacingBefore
+    summary: The distance between the top of paragraph and the beginning of its text content.
+    description: |
+      This property contains the space (measured in points) between the top of paragraph and 
+      the beginning of its text content. The default value of this property is 0.0.
+    type: Number
+
+  - name: lineHeightMultiple
+    summary: The line height multiple.
+    description: |
+      The natural line height of the receiver is multiplied by this factor (if positive) 
+      before being constrained by minimum and maximum line height. The default value of 
+      this property is 0.0.
+    type: Number
```

Comment:
The docs here don't do a good job describing what `lineHeightMultiple`, `maximumLineHeight`, and `minimumLineHeight` do. Nor why you would want to use them.

I assume you would want to use `minimumLineHeight` in case you use mixed font sizes and you want all lines to have the same height as the largest font size. That's the only use-case I can think of.

But why would you want to use `maximumLineHeight`? Would it clip text that exceed this height? If there is answer to this, then we should document it.

Also, I have no clue what `lineHeightMultiple` is supposed to do from reading the description... nor why I would want to use it. If there is a valid use-case, then let's document it.

Reply:
Also, I assume a value of zero for `maximumLineHeight` means do not apply a max, right? Otherwise text would always be clipped. We should document that.

I assume the same is true for `lineHeightMultiple`? A value of zero means don't multiply?